handbook/people-ops/travel.md

```diff
@@ -1,98 +1,166 @@
+-   This budget can be used for travel *(airfare, trains, etc.)*, lodging *(hotels, rental homes, etc.)*, events, meals and entertainment.
+-   Teams are expected to use discretion about what events are appropriate for team outings. Be inclusive of all of the groups interests.
+-   Reserve plenty of time for just hanging out, touring around, and having fun. Quality time with eachother is important.
+-   If one individual is a member of multiple teams, they will receive the same amount for each of their teams.
+-   The budget can be used for online team bonding events if the team prefers. See our [**Remote-friendly team bonding document**](https://docs.google.com/document/d/1tvWXo0nu49yhn7f0lWIHBwCKfTvoSftbN2j7um0joWI/edit?ts=60906da4) for ideas.
+
+## [](https://about.sourcegraph.com/handbook/people-ops/travel#individual-travel "#individual-travel")3. Individual travel
+
+Each teammate has one fully-paid trip per year to meet up with an individual or a group, and this can include anyone in the company.
+
+ - The purpose of defining this as one “trip” (instead of providing a set dollar budget) is to avoid punishing teammates that live far away from other teammates, and who would therefore incur higher airfare costs.
+ - This trip is  _in addition_  to the other benefits mentioned on this page.
+
+## [](https://about.sourcegraph.com/handbook/people-ops/travel#professional-development-and-education "#professional-development-and-education")4. Business/customer travel
+
+For teammates with business-related travel requirements, trips will be fully paid pending manager/director approval in addition to above budgets. We would include the following types of meetups:
 
-For any additional questions, please reach out to #finance or #people-ops.
+ - Customer Champion meetups 
+ - Executive meetups (no approval required)
+ - Marketing conferences 
+ - Mandatory team trainings
+ -  Sales club celebrations 
+ - Quarterly Regional Sales Kickoff (Sales, Sales Ops, CE included)
 
+## [](https://about.sourcegraph.com/handbook/people-ops/travel#professional-development-and-education "#professional-development-and-education")5. Professional development and educational travel
 
-## Company-wide events
+Each teammate has $1,000 to spend on professional development, which could include attendance fees and travel to conferences, educational trainings, university classes, etc. as outlined on our  [benefits page](https://about.sourcegraph.com/handbook/people-ops/benefits-and-perks#professional-development).
 
-Sourcegraph generally holds two company-wide events each year, each fully covered by the company:
+# Budget best practices and additional spending categories
 
-- Winter party (typically in San Francisco)
-- Spring/summer event
+## Hotel accomodations:
 
-## Team events
+We advise spending within reason for the location you are visiting, with a $400 USD per night maximum spend limit. 
```

Comment:
At my previous employer, we ended up developing more fine grained policies based on the cost of living/accommodation in certain high cost cities. ($400 a night can be problematic in the Bay Area; meanwhile you've probably just rented an entire town in the Western Australian wheatbelt.) We may want to consider doing the same here eventually, as it helps manage expectations.